fuzzylboxstats.py

```diff
@@ -330,11 +336,12 @@ def run_single(filename, main, sub, exprs, mrepl, srepl, msample=None, ssample=N
         r = fuz.run(msample, ssample)
     except Exception, e:
         # We only care about files that parse initially
+        print("Error in", filename)
         import traceback
```

Comment:
Should we just `raise e` at this point to simplify things?

Reply:
If we do that the experiment will stop completely. I used to just print `s` here, but put in the traceback to see what going on. I should probably remove it completely.

Reply:
Ah, I see. OK, removing it is probably a reasonable approach.